Rakefile

```diff
@@ -14,3 +14,17 @@ desc 'Count the number of hash types'
 task :count do
   puts File.read('data/prototypes.json').scan('"extended"').count
 end
+
+# Define a task for installing the man page
+desc 'Install the man page'
+task :install_manpage do
+  # Set the source and destination paths
+  manpage_source = 'man/haiti.1'
+  manpage_destination = '/usr/share/man/man1/'  # Adjust this path to match your system's man page directory
+
+  # Copy the man page to the destination
+  sh "cp #{manpage_source} #{manpage_destination}"
```

Comment:
The rake task is not targeted for the end user but for the dev, so we don't want to install the man page on the system but to generate the man page from the ronn file.

Something like

```ruby
  sh "ronn --roff man/haiti.ronn"
```

The deployment of the man page in `/usr/share/man/man1/` will be the role of the distro package (copying from them gem folder to the man db).

Reply:
So, the entire rake task `install_manpage`, would just be  ` sh "ronn --roff man/haiti.ronn"`?

Reply:
Yes

Reply:
Updated!